cilantro_audit/admin_page.py

```diff
@@ -0,0 +1,11 @@
+from kivy.uix.screenmanager import Screen
+
+from cilantro_audit.constants import COMPLETED_AUDITS_LIST_PAGE
+
+
+class AdminPage(Screen):
+    pass
+
+    def load_completed_audits_list_page(self, root, widget):
+        root.manager.current = COMPLETED_AUDITS_LIST_PAGE
+        widget.load_completed_audits()
```

Comment:
Probably not needed anymore since you resolved the issue in the .kv file via
`root.manager.get_screen(THIS_PAGE).load_completed_audits()`

Reply:
^ Nice catch. I forgot to remove that part since we were working on it.